studio/pages/project/[ref]/reports/database.tsx

```diff
@@ -51,9 +54,12 @@ const DatabaseUsage = observer(() => {
   })
   const databaseSize = data?.result[0].db_size ?? 0
 
-  const databaseSizeUsageRatio = databaseSize / databaseSizeLimit
-  const sizeIsApproaching = databaseSizeUsageRatio >= USAGE_APPROACHING_THRESHOLD
-  const sizeIsExceeded = databaseSizeUsageRatio >= 1
+  const databaseSizeUsageRatio = databaseSize / databaseUsageLimit
+  const limitIsApproaching = databaseSizeUsageRatio >= USAGE_APPROACHING_THRESHOLD
+  const limitIsExceeded = databaseSizeUsageRatio >= 1
+
+  const diskSize = usage?.disk_volume_size_gb ?? 0
+  const diskSizeUsageRatio = databaseSize / diskSize
```

Comment:
hmm this might error out if `diskSize` defaults to 0

perhaps `const diskSizeUsageRatio = diskSize !== 0 ? databaseSize / diskSize : 0` ?